test/e2e_node/container_lifecycle_pod_construction.go

```diff
@@ -250,21 +252,28 @@ func (o containerOutputList) findIndex(name string, command string, startIdx int
 
 // parseOutput combines the termination log from all of the init and regular containers and parses/sorts the outputs to
 // produce an execution log
-func parseOutput(pod *v1.Pod) containerOutputList {
+func parseOutput(ctx context.Context, f *framework.Framework, pod *v1.Pod) containerOutputList {
 	// accumulate all of our statuses
 	var statuses []v1.ContainerStatus
 	statuses = append(statuses, pod.Status.InitContainerStatuses...)
 	statuses = append(statuses, pod.Status.ContainerStatuses...)
+
 	var buf bytes.Buffer
 	for _, cs := range statuses {
-		// If the container is terminated but the reason is ContainerStatusUnknown,
-		// it means that the kubelet has overwritten the termination message. Read
-		// the LastTerminationState instead.
-		if cs.State.Terminated != nil && cs.State.Terminated.Reason != "ContainerStatusUnknown" {
-			buf.WriteString(cs.State.Terminated.Message)
-		} else if cs.LastTerminationState.Terminated != nil {
-			buf.WriteString(cs.LastTerminationState.Terminated.Message)
+		if cs.State.Running == nil && cs.State.Terminated == nil {
+			continue
 		}
+		log, err := e2epod.GetPodLogs(ctx, f.ClientSet, f.Namespace.Name, pod.Name, cs.Name)
+		framework.ExpectNoError(err)
+		buf.WriteString(log)
+		//		// If the container is terminated but the reason is ContainerStatusUnknown,
```

Comment:
Remove before merge, assuming you left it in temporarily just to ensure all tests pass.